.github/workflows/build-monorepo.yml

```diff
@@ -0,0 +1,76 @@
+name: Test build in monorepo
+on:
+  workflow_call:
+    inputs:
+      runs_on:
+        required: true
+        type: string
+      os:
+        required: true
+        type: string
+      concurrency_group:
+        required: true
+        type: string
+      is_hoisted:
+        required: true
+        type: boolean
+
+jobs:
+  build:
+    name: Build monorepo with Reanimated
+    runs-on: ${{ inputs.runs_on }}
+    concurrency:
+      group: ${{ inputs.concurrency_group }}
+      cancel-in-progress: true
+    steps:
+      - name: Create monorepo
+        run: |
+          mkdir monorepo
+          cd monorepo
+          echo '{"name":"rnos-monorepo-tester","version":"1.0.0","license":"MIT","private":true,"workspaces":{"packages":["RootApp","packages/PackageApp", "AppA", "AppB"],"nohoist":["**/react","**/react-dom","**/react-native","**/react-native/**","**/react-native-codegen","**/react-native-dev-menu"]}}' > package.json
+          yarn
+          mkdir packages
+      - name: Create RootApp
```

Comment:
I'm wondering if we can name these apps `HoistedReanimatedApp` and `NonHoistedReanimatedApp` so that we don't need to remember if PackageApp is hoisted or not? BTW What's the better word for "non-hoisted"?